alluxio-python-library/alluxio/alluxio_file_system.py

```diff
@@ -278,6 +306,40 @@ def _create_session(self, concurrency):
         session.mount("http://", adapter)
         return session
 
+    def _load_file(self, worker_host, path):
+        try:
+            response = self.session.get(
+                self.LOAD_SUBMIT_URL_FORMAT.format(
+                    worker_host=worker_host,
+                    http_port=self.http_port,
+                    path=path,
+                ),
+            )
+            response.raise_for_status()
+            while True:
+                response = self.session.get(
+                    self.LOAD_PROGRESS_URL_FORMAT.format(
+                        worker_host=worker_host,
+                        http_port=self.http_port,
+                        path=path,
+                    ),
+                )
+                response.raise_for_status()
+                content = response.content
+                if b"SUCCEEDED" in content:
+                    print(content.decode('utf-8'))
+                    return True
+                if b"FAILED" in content:
+                    print(content.decode('utf-8'))
```

Comment:
can we throw exception to end user with the FAILED reasons (content)

Reply:
Sure.